websites/R/Reddit/presence.ts

```diff
@@ -1,20 +1,21 @@
-let presence: Presence;
-
-function setPresence(): void {
-	if (location.pathname.includes("/r/netflix"))
-		presence = new Presence({ clientId: "869992823854870588" });
-	else presence = new Presence({ clientId: "609183409440555018" });
+enum PresenceClients {
+	Reddit = "609183409440555018",
+	RedditNetflix = "869992823854870588",
 }
-
-setPresence();
-
-let subReddit: string,
+let presence = new Presence({ clientId: PresenceClients.Reddit }),
+	subReddit: string,
 	postTitle: string,
 	username: string,
 	nickname: string,
 	rpanTitle: string,
 	strings: Awaited<ReturnType<typeof getStrings>>,
-	oldLang: string = null;
+	oldLang: string = null,
+	containsNetflix: boolean;
+
+const enum Assets {
+	Logo = "https://cdn.rcd.gg/PreMiD/websites/R/Reddit/assets/logo.png",
+	NetflixLogo = "https://i.imgur.com/Aw5rIOI.gif",
```

Comment:
```suggestion
	NetflixLogo = `https://i.imgur.com${"/Aw5rIOI.gif"}`,
```

Reply:
why???

Reply:
There seems to be a bug with resizing/uploading gifs apparently

Reply:
Although, there are examples where there are no problems:
* https://cdn.rcd.gg/PreMiD/websites/N/Netflix/assets/0.gif
* https://cdn.rcd.gg/PreMiD/websites/B/BeatLeader/assets/81.gif
* https://cdn.rcd.gg/PreMiD/websites/D/Discovery/assets/0.gif
* etc

@Bas950

Reply:
The Netflix and Discovery ones I had to manually override
I mean we can test if it works with this one...